simcon_project/templates/feedback/view_feedback.html

```diff
@@ -19,10 +19,9 @@
 
```

Comment:
{% block content %} is in a weird place in this file and i think it's keeping the right title from showing up. also don't think we need the bootstrap import here.